src/query/interpret/eval.hpp

```diff
@@ -1115,5 +1173,6 @@ int64_t EvaluateInt(ExpressionEvaluator *evaluator, Expression *expr, const std:
 
 std::optional<size_t> EvaluateMemoryLimit(ExpressionVisitor<TypedValue> &eval, Expression *memory_limit,
                                           size_t memory_scale);
+// std::optional<size_t> EvaluateMemoryLimit(ExpressionEvaluator *eval, Expression *memory_limit, size_t memory_scale);
```

Comment:
Do we need this comment?

Reply:
Just a leftover from merging main into this branch, removed.